app/webpacker/components/Persons/Person/PersonalRecords.jsx

```diff
@@ -0,0 +1,220 @@
+import React from 'react';
+import {
+  Header, Icon, Popup, Table,
+} from 'semantic-ui-react';
+import I18nHTMLTranslate from '../../I18nHTMLTranslate';
+import I18n from '../../../lib/i18n';
+import { events } from '../../../lib/wca-data.js.erb';
+import EventIcon from '../../wca/EventIcon';
+import { rankingsPath } from '../../../lib/requests/routes.js.erb';
+import { formatAttemptResult } from '../../../lib/wca-live/attempts';
+
+function isOddRank(rank) {
```

Comment:
Maybe `isCounterintuitiveRank` or `rankNeedsExplanation` or something? Since odd means something else already for numbers.